runtime/moonbase/src/relay_indices.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019-2022 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+use pallet_xcm_transactor::relay_indices::*;
+
+/// Westend pallet and extrinsic indices
+pub const WESTEND_RELAY_INDICES: RelayChainIndices = RelayChainIndices {
```

Comment:
I am unsure whether this is the right place where these should live in relay-encoder...what do you think?

Reply:
agree, moved to `moonbeam-relay-encoder`.